lib/model/rwfolder.go

```diff
@@ -780,13 +778,9 @@ func (f *rwFolder) renameFile(source, target protocol.FileInfo) {
 	from := filepath.Join(f.dir, source.Name)
 	to := filepath.Join(f.dir, target.Name)
 
-	if f.versioner != nil {
```

Comment:
I tend to put this into the versioners as `preserveOldFile` method. What do you think?

P.S. It is very hard to concentrate at almost midnight ;)